pyiron_base/job/jobtype.py

```diff
@@ -109,7 +110,7 @@ def convert_str_to_class(job_class_dict, class_name):
         )
 
 
-class JobCreator(metaclass=Singleton):
+class JobFactory(PyironFactory):
```

Comment:
I thought it is required to be a singleton for the updating for the job list to work when we import `pyiron` or `pyiron_contrib`.

Reply:
With the Singleton requirement the tests failed, without it they passed, so I just got rid of it 😛 I will do the integration test to make sure importing a sub-package (alt-package?) still works.

Reply:
@jan-janssen it seems to work fine. With this branch and the contrib master, the following worked fine:

```python
from pyiron import Project
import pyiron_contrib

# Run a protocol
job = pr.create.job.Lammps('lmp_job')
job.structure = pr.create.structure.ase_bulk('Al', cubic=True)
job.potential = job.list_potentials()[0]
job.save()

prot = pr.create.job.ProtocolMD('prot')
prot.input.structure = job.structure.copy()
prot.input.temperature = 300
prot.input.n_steps = 10
prot.input.ref_job_full_path = job.path
prot.run()

another_job = pr.create.job.Lammps('another_lmp_job')
another_job.structure = job.structure.copy()
another_job.run()
```

i.e. jobs of both the base and imported modules are both creating and running fine.

If you're satisfied, please go ahead and merge 😄